tests/Backend/Redshift/OptimizeTest.php

```diff
@@ -32,7 +32,7 @@ public function testOptimize(): void
             ]
         );
 
-        $this->_client->apiPost("tables/{$tableId}/optimize");
+        $this->_client->apiPostJson("tables/{$tableId}/optimize");
```

Comment:
Tady jsi to asi mohl nechat, pac se neposilaji zadny parametry, takze ani JSON. Ale je to fuk, bude to fungovat taky...